ui/raidboss/data/05-shb/etc/the_diadem.js

```diff
@@ -20,13 +20,15 @@ export default {
       id: 'Diadem Found Gather Point',
       netRegex: NetRegexes.gameLog({ line: 'You sense a grade .* clouded (?:mineral deposit|rocky outcrop|mature tree|lush vegetation patch).*?', capture: false }),
       netRegexDe: NetRegexes.gameLog({ line: 'Du hast einen? verhüllten? (?:Nutzbaum|Vegetation|Steinbruch|Lagerstätte)? der Güte.*?', capture: false }),
+      netRegexFr: NetRegexes.gameLog({ line: 'Vous détectez une? (?:filon de minerai|carrière de pierre|arbre mature|carré de végétation) évanescente? de niveau 80 du côté (?:est|sud|ouest|nord|nord-est|sud-est|nord-ouest|sud-ouest).*?', capture: false }),
```

Comment:
is correct ?

Reply:
I think it is ok, but didn't test all of them in-game.
It takes long time to wait for a gather point in diadem.

Reply:
Yes, i know.